yarn.lock

```diff
@@ -17351,7 +17438,7 @@ next-api-middleware@^1.0.1:
   dependencies:
     debug "^4.3.2"
 
-next-auth@^4.10.3:
+next-auth@^4.10.3, next-auth@^4.3.3:
```

Comment:
We have a version mismatch here @agustif let's sync all submodules before committing the `yarn.lock` file. Or since we're not adding more dependencies, discard it altogether

Reply:
yes, removed all yarn.lock changes since we're not adding a new dep `lru-cache` and its types anymore, good catch thanks zomars